tests/kola/networking/no-default-initramfs-net-propagation/test.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -xeuo pipefail
+
+# With pure network defaults no networking should have been propagated
+# from the initramfs. This test tries to verify that is the case.
+# https://github.com/coreos/fedora-coreos-tracker/issues/696
+
+ok() {
+    echo "ok" "$@"
+}
+
+fatal() {
+    echo "$@" >&2
+    exit 1
+}
+
+if ! journalctl -t coreos-teardown-initramfs | \
+       grep 'info: skipping propagation of default networking configs'; then
```

Comment:
FWIW you can do `journalctl -t coreos-teardown-initramfs --grep 'info: skipping propagation of default networking configs` too.  (It's a bit more efficient since it can be server side searching)

Reply:
nice, didn't know about that. Thanks!